packages/toolkit/src/query/react/buildHooks.ts

```diff
@@ -775,21 +788,34 @@ export function buildHooks<Definitions extends EndpointDefinitions>({
         }
       }, [])
 
-      return useMemo(
-        () => ({
-          /**
-           * A method to manually refetch data for the query
-           */
-          refetch: () => {
-            if (!promiseRef.current)
-              throw new Error(
-                'Cannot refetch a query that has not been started yet.'
-              )
-            return promiseRef.current?.refetch()
+      return {
```

Comment:
I don't understand the entire code, but `use` will attach `.value` prop, so you want to keep `useMemo` maybe?

Reply:
Hmm, good point, that might be very tricky.

The return value of this hook will be spread in yet another hook, so having a stable value here will not help a lot - and that other value can be memoized, but will change more often since it has other "changing values" in it.

We would probably have to manage `.value` on our own in that context 🤔